tests/CMakeLists.txt

```diff
@@ -17,3 +17,7 @@ add_test(test_dimr_bmi test_dimr_bmi)
 add_executable(test_io_layer_distribution test_io_layer_distribution.c)
 target_link_libraries(test_io_layer_distribution PRIVATE test_dependencies)
 add_test(test_io_layer_distribution test_io_layer_distribution)
+
+add_executable(test_load_csv test_load_csv.c)
+target_link_libraries(test_load_csv PRIVATE test_dependencies)
+add_test(NAME test_load_csv COMMAND test_load_csv WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
Oh nice, perhaps you can also use the NAME and COMMAND keywords for the rest (for consistency and readability)

Reply:
Done.